### PR TITLE
Public Cloud: Make sure the openqa_var_SERVER tag does not contain slash

### PR DESCRIPTION
**Pull request by pdostal**
- Verification run: [openqa.suse.de](https://openqa.suse.de/tests/11609478), [pdostal-server.suse.cz](https://pdostal-server.suse.cz/tests/4712)

### CODE REVIEW

**Review by ricardobranco777 — APPROVED** [review]
LGTM

File: lib/publiccloud/provider.pm [hunk 1]
```diff
@@ -623,11 +623,13 @@ c<{"key": "value"}> where c<value> must be a string.
 sub terraform_param_tags
 {
     my ($self) = @_;
+    my $openqa_var_server = get_var('OPENQA_URL', get_var('OPENQA_HOSTNAME'));
+    $openqa_var_server =~ s@^https?://|/$@@gm;
```
Comment (grisu48) [inline lib/publiccloud/provider.pm:627]: Please add a comment here for our future selves that will be thankful for it!

**Review by grisu48 — APPROVED** [review]

File: lib/publiccloud/provider.pm (hunk 1 above)
Comment (asmorodskyi) [inline lib/publiccloud/provider.pm:627]: keep in mind that `$openqa_var_server` can be just FQDN . I assume this will not be a problem here , right ?

Comment (pdostal) [reply lib/publiccloud/provider.pm:628]: I hope not.
image

**Pull request merged**